packages/@webex/widgets/package.json

```diff
@@ -33,8 +33,8 @@
   "dependencies": {
     "@webex/component-adapter-interfaces": "^1.30.5",
     "@webex/components": "1.275.2",
-    "@webex/sdk-component-adapter": "1.112.12",
-    "webex": "2.60.4"
+    "@webex/sdk-component-adapter": "1.112.11",
+    "webex": "2.60.2"
```

Comment:
This is resetting Parimala's change. Probably a conflict resolution issue. Please check

Reply:
I missed this. Reverted this change